### DISCUSSION TITLE
Add a line for the current Date

### DISCUSSION THREAD

**Discussion by TomassPasc** [deejay-hub/timeline-lwc#248] [Q&A] [answered]
Hello everyone, I try to put a line for the current date but I'm stuck.
I tried several ways but I can't get a dynamic line. For example I tried this in the timelineCanvas() method, a line appears but remains fixed on the canvas:

              let currentDate = new Date();
              timelineCanvas.append("line")
              .attr("class", "line")
              .attr("x1", timelineCanvas.x(currentDate))
              .attr("y1", 0)
              .attr("x2", timelineCanvas.x(currentDate))
              .attr("y2", 1600)
              .style("stroke", "steelblue")
              .style("stroke-width", "2px");

Maybe someone has an idea?

**Comment by deejay-hub** [comment, marked as answer]
You will need to redraw the line when the minimap zoom control is moved. You'll have to add the code to the timelineCanvas.redraw method.

```
timelineCanvas.selectAll('[class~=timeline-canvas-current-date]').remove();

            let cd = timelineCanvas.append('g')
                .attr('class', 'timeline-canvas-current-date')
                .attr('transform', 'translate(' + timelineCanvas.x(currentDate) + ')' );
            
            cd.append("line")
                .style("stroke", "steelblue")
                .style("stroke-width", "1px")
                .attr("y2", timelineHeight);
```

**Comment by deejay-hub** [comment]
We'll add this functionality in the next release.
